src/util.js

```diff
@@ -1,5 +1,14 @@
 /* Common utilities */
 WaveSurfer.util = {
+    requestAnimationFrame: (
+        window.requestAnimationFrame ||
+        window.webkitRequestAnimationFrame ||
+        window.mozRequestAnimationFrame ||
+        window.oRequestAnimationFrame ||
+        window.msRequestAnimationFrame ||
+        function (callback, element) {window.setTimeout(callback, 1000 / 60);}
```

Comment:
Thanks! Could you add spaces inside the anonymous function? And no need for `window`, `setTimeout` is a global also in Node.